src/backend/columnar/sql/columnar--10.2-2--10.2-3.sql

```diff
@@ -10,6 +10,17 @@
 --
```

Comment:
What Citus versions has this been released in? Do we need to backport this? What would happen if any of the previously released Citus versions is still used somewhere. Guess that it will fail directly during the `ALTER EXTENSION citus UPDATE;` ? Which probably would suffice to have a patch release for 10.2, 11.0 and 11.1?